File: Beginner/Exer. 1008/main.c
#include <stdio.h>

int main(int argc, char *argv[]) {
    int number, worked;
    double received;
    
    scanf("%d", &number);
    scanf("%d", &worked);
    scanf("%lf", &received);
    
    printf("NUMBER = %d\n", number);
    printf("SALARY = U$ %.2lf\n", (received * worked));
    return 0;
}

File: Beginner/Exer. 1008/main.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int number = scanner.nextInt();
        int worked = scanner.nextInt();
        double received = scanner.nextDouble();

        scanner.close();

        System.out.println("NUMBER = " + number);
        System.out.printf("SALARY = U$ %.2f\n", worked * received);
    }
}